Servers/UI/OJS.Servers.Ui/ClientApp/config-overrides.js

```diff
@@ -38,10 +38,11 @@ const addMonacoPlugin = (config) => ({
     ],
 });
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This is a left over that should persist for now.
Current it breaks the rebuilding of the UI, but is necessary for measuring the build performance